bin/exampleserver

```diff
@@ -52,24 +52,6 @@ var port = argv.p;
 // Attach the sharejs REST and Socket.io interfaces to the server
 var server = sharejs.server.attach(app, options);
 
-var renderer = require('../examples/_static');
-app.get('/static/:docName', function(req, res, next) {
-  var docName;
-  docName = req.params.docName;
-  renderer(docName, app.model, res, next);
-});
-
-var wiki = require('../examples/_wiki');
-app.get('/wiki/?', function(req, res, next) {
-  res.writeHead(301, {location: '/wiki/Main'});
-  res.end();
-});
-app.get('/wiki/:docName', function(req, res, next) {
-  var docName;
-  docName = req.params.docName;
-  wiki(docName, app.model, res, next);
-});
-
```

Comment:
When executing `./bin/exampleserver` I get the "Invalid or unsupported database type: 'redis'" error message.

It seems the default server for the example server is set to "redis": https://github.com/conversation/ShareJS/blob/22997a307926a70ff4cb6c5972fa2d944bf17999/bin/exampleserver#L41-L45

After I updated the `bin/exampleserver` file to have the `options.db = { type: 'none' }`, it worked just fine, with in memory persistence.

Reply:
Ah yep, I think because I didn't have redis installed it was just skipping over that part for me, have removed those lines of code so it should default to memory in all cases now.